stories/core/Modal.stories.tsx

```diff
@@ -11,6 +11,7 @@ import { Modal, ModalButtonBar } from '../../src/core/Modal';
 import { ModalProps } from '../../src/core/Modal/Modal';
 import { useEffect, useState } from '@storybook/addons';
 import { CreeveyMeta } from 'creevey';
+import ModalContent from '../../src/core/Modal/ModalContent';
```

Comment:
Move this import to line 10. Or you can even import everything from `src/core`.

Reply:
Done